data/ext/pending/issue-1423.ttl

```diff
@@ -30,3 +45,46 @@
     :source <https://github.com/schemaorg/schemaorg/issues/1423> ;
     rdfs:comment "Human-readable terms of service documentation." .
 
+:endpointUrl a rdf:Property ;
+    rdfs:label "endpointUrl" ;
+    :category "issue-1423" ;
+    :domainIncludes :WebAPI ;
+    :isPartOf <http://pending.schema.org> ;
+    :rangeIncludes :EntryPoint,
+        :URL ;
+    :source <https://github.com/schemaorg/schemaorg/issues/1423> ;
+    rdfs:comment "The root location or primary endpoint of the API." .
+
+:conformsTo a rdf:Property ;
```

Comment:
It's been suggested (uk govt folks - I'll ask them to share the comment) that we add a more general 'conformsTo' to Schema.org, broadly in line with the Dublin Core term. I think we should do that - but the definition would need to be more general than APIs. I think it would still be applicable here.